src/qililab/qprogram/qblox_compiler.py

```diff
@@ -301,18 +311,13 @@ def _handle_reset_phase(self, element: ResetPhase):
 
     def _handle_set_gain(self, element: SetGain):
         convert = QbloxCompiler._convert_value(element)
-        gain_0 = (
-            self._buses[element.bus].variable_to_register[element.gain_path0]
-            if isinstance(element.gain_path0, Variable)
-            else convert(element.gain_path0)
-        )
-        gain_1 = (
-            self._buses[element.bus].variable_to_register[element.gain_path1]
-            if isinstance(element.gain_path1, Variable)
-            else convert(element.gain_path1)
+        gain = (
+            self._buses[element.bus].variable_to_register[element.gain]
+            if isinstance(element.gain, Variable)
+            else convert(element.gain)
         )
         self._buses[element.bus].qpy_block_stack[-1].append_component(
-            component=QPyInstructions.SetAwgGain(gain_0=gain_0, gain_1=gain_1)
+            component=QPyInstructions.SetAwgGain(gain_0=gain, gain_1=gain)
         )
```

Comment:
why are both gains the same?

Reply:
Because on IQ buses the gain of both paths must be the same:

![image](https://github.com/qilimanjaro-tech/qililab/assets/3442573/a620ff09-92e4-49d9-8ddb-c5e75a2a7e78)